shared/switches.py

```diff
@@ -1,46 +0,0 @@
-"""Module to create a singleton class to track all switches. After being
-loaded by the main napp, this class can be reached just using
-Switches().desired_method()
-
-For example:
-
-from switches import Switches
-
-list_of_switches = Switches().get_switches()
-switch_01 = Switches().get_switch("00:00:00:00:00:00:00:01")
-"""
-
-from napps.amlight.coloring.shared.singleton import Singleton
```

Comment:
@rmotitsuki since you're at it, it seems that this `napps.amlight.coloring.shared.singleton` module isn't being used either, maybe it could be removed? Could you double check?